tests/test_pipelines_fill_mask.py

```diff
@@ -186,7 +186,7 @@ def run_pipeline_test(self, model, tokenizer, feature_extractor):
             ],
         )
 
-        outputs = fill_masker([f"This is a {tokenizer.mask_token}", f"Another {tokenizer.mask_token}"])
+        outputs = fill_masker([f"This is a {tokenizer.mask_token}", f"Another {tokenizer.mask_token} great test."])
```

Comment:
why is this added here?

Reply:
But the test only checks for type so should be fine

Reply:
Because the length was too small I guess.

I don't like having this change in there though.